applications/API/urls.py

```
@@ -0,0 +1,35 @@
+)
+from applications.API.endpoints import CreateInsultView, InsultByCategoryEndpoint,ListCategoryView, RandomInsultView,InsultDetailsEndpoints, InsultListView
+
+
+urlpatterns = [
+    # API documentation/schema endpoints
+    path("schema/", SpectacularAPIView.as_view(), name="schema"),
+    path("swagger/", SpectacularSwaggerView.as_view(), name="swagger"),
+    path("redoc/", SpectacularRedocView.as_view(), name="redoc"),
+    # Categories (static-ish)
+    path("api/categories/", ListCategoryView.as_view(), name="list_categories"),
+    # Insults – collection first
```

Ruff (0.12.2)

Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)